source/broadband/webpa_rbus.c

```diff
@@ -124,3 +129,51 @@ rbusError_t clearTraceContext()
 		WalError("Rbus not initialized in clearTraceContext funcion\n");
         }
 }
+
+static void CloudConnOnlineCallbackHandler(
+    rbusHandle_t handle,
+    rbusEvent_t const* event,
+    rbusEventSubscription_t* subscription)
+{
+
+    rbusValue_t newValue = rbusObject_GetValue(event->data, "value");
+    
+    int incoming_value = rbusValue_GetInt32(newValue);
+
+    WalInfo("Received cloud online callback event %s\n", event->name);
+
+    if(incoming_value)
+    {
+	WalPrint("Received cloud connection online event\n");
+	pthread_mutex_lock (get_global_sync_mutex());
+	//Triggering cloud connection online event for retrying sync notification.
```

Comment:
this comment is not correct. we are not triggering cloud connection online event . this is just a signal to do sync retry since cloud connection online event is received